examples/slash_commands/param.py

```diff
@@ -165,6 +165,20 @@ async def autocomplete(
     ...
 
 
+# From version 2.2.0
+# it's possible to create autocomplete options with the decorator @slash_command.autocomplete()
+
+
+@bot.slash_command()
+async def languages(inter: disnake.ApplicationCommandInteraction, language: str):
+    ...
+
+
+@languages.autocomplete("language")
+async def language_autocomp(inter: disnake.ApplicationCommandInteraction, string: str):
+    return ["Python", "JavaScript", "TypeScript", "Java", "Rust", "Lisp", "Elixir"] 
```

Comment:
The whole point of autocomplete here isn't really shown, I think you can do something like this:
```py
return [lang for lang in langs if string in lang.lower()]
```

Reply:
My bad, sorry. 
Are type hints necessary here?

Reply:
Type hints are nice and helpful

Reply:
But please in moderation. Looking at you LEv145.